internal/rbd/rbd_util.go

```diff
@@ -800,18 +803,36 @@ func deleteSnapshot(ctx context.Context, vol *rbdVolume, pOpts *rbdSnapshot, cr
 	return err
 }
 
-// nolint: interfacer
-func restoreSnapshot(ctx context.Context, pVolOpts *rbdVolume, pSnapOpts *rbdSnapshot, cr *util.Credentials) error {
-	var output []byte
+func cloneRbdImageFromSnapshot(ctx context.Context, pVolOpts *rbdVolume, pSnapOpts *rbdSnapshot, cr *util.Credentials) error {
```

Comment:
This is a nice chance on moving `cloneRbdImageFromSnapshot()` to a method of `rbdSnapshot`. Can be done as a cleanup later.